lib/srv/app/session.go

```diff
@@ -44,6 +45,10 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+const sessionChunkCloseTimeout = 2 * time.Minute
```

Comment:
Would there be any downside to increasing this to, say, an hour? To allow for long-running requests. Cause it would trigger only in the worst case IIUC.

Also, please add a comment here explaining the purpose of this timeout.